ga4gh/backend.py

```diff
@@ -192,6 +192,63 @@ def _getEnd(cls, variant):
         return variant.end
 
 
+class VariantAnnotationsIntervalIterator(IntervalIterator):
+    """
+    An interval iterator for annotations
+    """
+
+    def __init__(self, request, parentContainer):
+        super(VariantAnnotationsIntervalIterator, self).__init__(
+            request, parentContainer)
+        self.featureIds = set(self._request.feature_ids)
```

Comment:
We should have private variables here: ``self._featureIds`` and ``self._effects``.